code/drasil-gool/GOOL/Drasil/LanguageRenderer.hs

```diff
@@ -163,6 +170,9 @@ bodyDocD bs = vibcat blocks
   where blocks = filter notNullBlock bs
         notNullBlock b = not $ isEmpty b
 
+oneLinerD :: (RenderSym repr) => repr (Statement repr) -> repr (Body repr)
```

Comment:
One of the things I meant to mention on issue #1868 is that one way to factor things out is to provide `repr`-generic functions (like this one) which works in all cases, by dispatching its work to the right lower-level combinators.